PurchasesTests/Networking/BackendTests.swift

```diff
@@ -150,7 +150,7 @@ class BackendTests: XCTestCase {
                                               "is_restore": isRestore,
                                               "observer_mode": observerMode],
                                        headers: ["Authorization": "Bearer " + apiKey])
-        expect(self.httpClient.calls.count).to(equal(1))
+        expect(self.httpClient.calls.count).toEventually(equal(1))
```

Comment:
These `toEventually` became necessary as I added an OperationQueue to execute stuff instead of remaining on the same thread as the Backend tasks.